src/main/features/response/index.ts

```diff
@@ -34,6 +35,10 @@ export class Feature {
       AlreadyRespondedGuard.requestHandler
     )
     app.all(
+      /^\/case\/.+\/response\/(?![\d]+\/receiver|confirmation|full-admission|partial-admission|counter-claim|receipt).*$/,
```

Comment:
For me it looks like this regex matches all paths starting with `/case/<something>/response` except `/case/<something>/response/<some-digits>/receiver`. IF only linking receiver is supported to be excluded then there is no point in having `confirmation|full-admission|partial-admission|counter-claim|receipt` as a part of regex.

Reply:
the receiver isn't even under case...